ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -190,17 +191,27 @@ def __init__(
         self,
         session_management_client: SessionManagementClient,
         session_info: Sequence[session_management_service_pb2.SessionInformation],
+        multiplexer_session_info: Optional[
+            Sequence[session_management_service_pb2.MultiplexerSessionInformation]
+        ] = None,
         reserved_pin_or_relay_names: Union[str, Iterable[str], None] = None,
         reserved_sites: Optional[Iterable[int]] = None,
     ) -> None:
         """Initialize reservation object."""
         super().__init__(session_management_client)
-        self._grpc_session_info = session_info  # needed for unreserve
+        self._grpc_session_info = session_info  # required for unreserve
```

Comment:
This seems like [splitting hairs](https://idioms.thefreedictionary.com/splitting+hairs) but whatever, I guess "required" is ok too.